tests/wpunit/MenuConnectionQueriesTest.php

```diff
@@ -32,6 +32,7 @@ public function testMenusQueryById() {
 
 		$actual = do_graphql_request( $query );
 
+		// use --debug to view
```

Comment:
I feel like we should add this to the readme somewhere, instead of adding this to individual `codecept_debug()` calls.